app/styles/iconography.ts

```diff
@@ -1,7 +1,17 @@
-import { ViewStyle } from 'react-native';
+import { ViewStyle, TextStyle } from 'react-native';
 
 import * as Colors from './colors';
 import * as Spacing from './spacing';
+import * as Typography from './typography';
+
+export const smallIcon: ViewStyle = {
+  height: 30,
+  width: 30,
+  borderRadius: 100,
```

Comment:
perhaps set this to 100%?

Reply:
sadly border radius doesnt accept percentages.